simple-icons/simple-icons
# Add SVG icon for Cal

### Checklist

- [ ] I updated the JSON data in `_data/simple-icons.json`
- [ ] I optimized the icon with SVGO or SVGOMG
- [ ] The SVG `viewbox` is `0 0 24 24`

### Description
This pull request adds an SVG icon for Cal, The SVG icon is created with scalability and adaptability in mind, ensuring consistent display across various devices and screen sizes.

Changes Made:

Added a new SVG icon for Cal.
Set the viewBox attribute to "0 0 24 24" to ensure consistent scaling and positioning.
Ensured that the SVG is properly formatted and optimized for performance.

How to Verify:

Review the newly added SVG file named "cal.svg".
Open the SVG file in an SVG viewer or web browser to ensure it displays correctly.

Additional Notes:

The SVG icon can be utilized in web applications, mobile apps, or any other platform where SVG graphics are supported.
The viewBox attribute of "0 0 24 24" enables the SVG icon to scale seamlessly and adapt to different screen sizes and resolutions.
Please review the changes and provide feedback if any further adjustments are needed. Thank you!
